tests/test_contracts.py

```
@@ -289,12 +321,27 @@ def test_create_orion_vault(self, MockConfig):
         # Check deposit access control passed
         assert args[6] == ZERO_ADDRESS
 
+    @patch("orion_finance_sdk_py.contracts.OrionConfig")
+    @pytest.mark.usefixtures("mock_load_abi", "mock_env")
+    def test_create_orion_vault_manager_not_whitelisted(self, MockConfig, mock_w3):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_w3`

(ARG002)